src/middlewares/userValidation.js

```diff
@@ -0,0 +1,28 @@
+const Joi = require('joi');
+
+const schemaCreateUser = Joi.object({
+  email: Joi.string()
+    .email({
+      minDomainSegments: 2,
+      tlds: { allow: false },
+    })
+    .required(),
+  password: Joi.string().min(6).max(18).required(),
+  firstName: Joi.string().alphanum().min(2).required(),
```

Comment:
I think it is better to use string for the firstName abd lastName

Reply:
Done